pegasus_wrapper/key_value.py

```diff
@@ -1,28 +1,50 @@
 from pathlib import Path
-from typing import Iterable, Tuple
+from typing import Any, Iterable, Mapping, Optional, Tuple
 
 from attr import attrib, attrs
 from attr.validators import instance_of
 
 from vistautils.parameters import Parameters
-from vistautils.scripts import join_key_value_stores
+from vistautils.scripts import downsample_key_value_store, join_key_value_stores
```

Comment:
Once the version of vistautils is updated in this PR CI should be able to find the `downsmaple_key_value_store`